test/e2e/operator.go

```diff
@@ -0,0 +1,78 @@
+package e2e
+
+import (
+	"os"
+	"testing"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+)
+
+var regularTestCases = map[string]*SampleFiles{
```

Comment:
This list is missing the Kaniko and buildah test cases with custom context and dockerfile name. They were named `kaniko-custom-context-dockerfile` in the original code `buildah-custom-context-dockerfile`.

Reply:
Thanks, Sascha. I overlooked that part, now it's in place again.